src/com/owncloud/android/lib/resources/status/OCCapability.java

```diff
@@ -77,6 +78,7 @@ public OCCapability(){
         mVersionMicro = 0;
         mVersionString = "";
         mVersionString = "";
+        mServerName = "Nextcloud";
```

Comment:
That is only half-true. In strings.xml (of the library) you create an app_name and put "Nextcloud" there. Once included in a project it will be overwritten with the app's strings.xml, and this is the approach we should take.